sale_order_qty_change_no_recompute/tests/test_sale_order_qty_change.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrderQtyChange(TransactionCase):
```

Comment:
Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.